daemon/daemon.go

```diff
@@ -442,7 +442,17 @@ configRetry:
 		log.Infof("Starting the Typha connection")
 		err := typhaConnection.Start(context.Background())
 		if err != nil {
-			log.WithError(err).Fatal("Failed to connect to Typha")
+			retry := 0
+			for err != nil && retry < 10 {
```

Comment:
You should put a `time.Sleep(1 * time.Second)` in this loop to throttle it if the connection fails immediately.

Reply:
Made me think about the possiblity that each retry could take 1ms or 30s depending on the type of problem.  Maybe swtich to a time-based loop:
```
startTime := time.Now()
for err!= nil && time.Since(startTime) < 30 * time.Second {
}
```

Reply:
Ack